### PR TITLE
Add scratch-dir option to work_queue_submit_common

### PR DESCRIPTION
**Pull request by colinthomas-z80**
Also had to edit condor_submit_workers to not overwrite the submit_dir

### CODE REVIEW

**Review by btovar — CHANGES_REQUESTED** [review]
Looks good, please also update the manuals:

doc/man/m4/*_submit_workers.m4

and then force the update of the md files:

rm doc/manuals/man_pages/*submit_workers.md
cd doc
make
git add -u
etc.

**Comment by colinthomas-z80** [comment]
Added man page info

File: doc/man/m4/condor_submit_workers.m4 [hunk 1]
```diff
@@ -42,7 +42,7 @@ OPTION_ARG(E, worker-options, str)Extra options passed to work_queue_worker
 OPTION_ARG_LONG(cores, cores)Set the number of cores each worker should use (0=auto). (default=1)
 OPTION_ARG_LONG(memory, size)Manually set the amonut of memory (in MB) reported by this worker.
 OPTION_ARG_LONG(disk, size)Manually set the amount of disk (in MB) reported by this worker.
-
+OPTION_ARG_LONG(scratch-dir,path)Set the scratch directory location created on the local machine. (default=/tmp/USER-workers)
```
Comment (btovar) [inline doc/man/m4/condor_submit_workers.m4:45]: default=/tmp/${USER}-workers

File: doc/man/m4/pbs_submit_workers.m4 [hunk 2]
```diff
@@ -34,6 +34,7 @@ OPTION_ARG_SHORT(O, os)Set operating system string for the worker to report to m
 OPTION_ARG_SHORT(s, path)Set the location for creating the working directory of the worker.
 OPTION_ARG_SHORT(j)Use job array to submit workers.
 OPTION_ARG_SHORT(p, parameters)PBS qsub parameters.
+OPTION_ARG_LONG(scratch-dir,path)Set the scatch directory location created on the local machine. (default=/tmp/USER-workers)
```
Comment (btovar) [inline doc/man/m4/pbs_submit_workers.m4:37]: The default for all others is not in /tmp, correct? It is in the current worker directory, e.g.: default=${USER}-workers

**Review by btovar — CHANGES_REQUESTED** [review]

File: doc/man/m4/pbs_submit_workers.m4 (hunk 2 above)
Comment (colinthomas-z80) [reply doc/man/m4/pbs_submit_workers.m4]: Yes you are correct. I have update the docs again. Thanks!

**Review by btovar — APPROVED** [review]

**Comment by btovar** [comment]
@colinthomas-z80 looks good!

**Pull request merged**